common/faults/faults.h

```diff
@@ -0,0 +1,237 @@
+#define TEST_FAULT_2_PRIORITY 1
+#define TEST_FAULT_3_PRIORITY 2
+#define TEST_FAULT_4_PRIORITY 0
+//END AUTO PRIORITY DEFS
+
+//BEGIN AUTO MAX DEFS
+#define BATT_FLOW_MAX 63
+#define DRIVE_FLOW_MAX 63
+#define MAIN_COMM_MAX 1
+#define LV_DEAD_MAX 3200
+#define MOT_FRONT_OT_MAX 100
+#define WLSPD_L_MAX 4096
+#define WLSPD_R_MAX 4096
+#define DRIVELINE_COMM_MAX 1
+#define BMS_MAX 1
+#define BATT_OT_MAX 50
+#define TV_OFFLINE_MAX 1
+#define TEST_FAULT_1_MAX 1
+#define TEST_FAULT_2_MAX 1
+#define TEST_FAULT_3_MAX 1
+#define TEST_FAULT_4_MAX 123
+//END AUTO MAX DEFS
+
+//BEGIN AUTO MIN DEFS
+#define BATT_FLOW_MIN 46
+#define DRIVE_FLOW_MIN 46
+#define MAIN_COMM_MIN 0
+#define LV_DEAD_MIN 2000
+#define MOT_FRONT_OT_MIN 10
+#define WLSPD_L_MIN 0
+#define WLSPD_R_MIN 0
+#define DRIVELINE_COMM_MIN 0
+#define BMS_MIN 0
+#define BATT_OT_MIN 10
+#define TV_OFFLINE_MIN 0
+#define TEST_FAULT_1_MIN 0
+#define TEST_FAULT_2_MIN 0
+#define TEST_FAULT_3_MIN 0
+#define TEST_FAULT_4_MIN 5
+//END AUTO MIN DEFS
+
+//BEGIN AUTO LATCH DEFS
+#define BATT_FLOW_LATCH_TIME 10
+#define DRIVE_FLOW_LATCH_TIME 10
+#define MAIN_COMM_LATCH_TIME 10
+#define LV_DEAD_LATCH_TIME 10
+#define MOT_FRONT_OT_LATCH_TIME 10
+#define WLSPD_L_LATCH_TIME 10
+#define WLSPD_R_LATCH_TIME 10
+#define DRIVELINE_COMM_LATCH_TIME 10
+#define BMS_LATCH_TIME 10
+#define BATT_OT_LATCH_TIME 10
+#define TV_OFFLINE_LATCH_TIME 10
+#define TEST_FAULT_1_LATCH_TIME 10
+#define TEST_FAULT_2_LATCH_TIME 10
+#define TEST_FAULT_3_LATCH_TIME 10
+#define TEST_FAULT_4_LATCH_TIME 10
+//END AUTO LATCH DEFS
+
+//BEGIN AUTO UNLATCH DEFS
+#define BATT_FLOW_UNLATCH_TIME 10
+#define DRIVE_FLOW_UNLATCH_TIME 10
+#define MAIN_COMM_UNLATCH_TIME 10
+#define LV_DEAD_UNLATCH_TIME 10
+#define MOT_FRONT_OT_UNLATCH_TIME 10
+#define WLSPD_L_UNLATCH_TIME 10
+#define WLSPD_R_UNLATCH_TIME 10
+#define DRIVELINE_COMM_UNLATCH_TIME 10
+#define BMS_UNLATCH_TIME 10
+#define BATT_OT_UNLATCH_TIME 10
+#define TV_OFFLINE_UNLATCH_TIME 10
+#define TEST_FAULT_1_UNLATCH_TIME 10
+#define TEST_FAULT_2_UNLATCH_TIME 10
+#define TEST_FAULT_3_UNLATCH_TIME 10
+#define TEST_FAULT_4_UNLATCH_TIME 10
+//END AUTO UNLATCH DEFS
+
+//BEGIN AUTO SCREENMSG DEFS
+#define BATT_FLOW_MSG "battery pump flow brokey\0" 
+#define DRIVE_FLOW_MSG "battery pump flow brokey\0" 
+#define MAIN_COMM_MSG "main not talking\0" 
+#define LV_DEAD_MSG "LV dead, shut car off ASAP\0" 
+#define MOT_FRONT_OT_MSG "Front motors overheating\0" 
+#define WLSPD_L_MSG "wheelspeeed L broken\0" 
+#define WLSPD_R_MSG "wheelspeeed R broken\0" 
+#define DRIVELINE_COMM_MSG "Driveline offline\0" 
+#define BMS_MSG "BMS fault\0" 
+#define BATT_OT_MSG "Battery overheating\0" 
+#define TV_OFFLINE_MSG "TV offline\0" 
+#define TEST_FAULT_1_MSG "Test fault 1\0" 
+#define TEST_FAULT_2_MSG "Test fault 2\0" 
+#define TEST_FAULT_3_MSG "Test fault 3\0" 
+#define TEST_FAULT_4_MSG "Test fault 4\0" 
+//END AUTO SCREENMSG DEFS
+
+//asdf
+
+
+//BEGIN AUTO ENUM DEFS
+typedef enum {
+	MAIN_MODULE = 0,
```

Comment:
These numbers were already defined in fault_nodes.h

Reply:
Will change, this enum isn't needed anymore